src/pages/my/FromCreatePage.tsx

```
@@ -6,25 +6,53 @@ import type { CreateFrom } from '@/types/from';
+import { useFromList } from '@/hooks/queries/useFromList';
 
 export default function FromCreatePage() {
   const navigate = useNavigate();
   const toast = useToast();
   const [input, setInput] = useState('');
   const createFromMutation = useCreateFrom();
 
+  const { data: fromList = [] } = useFromList();
+
+  const normalizeFromName = (name: string) =>
+    name.trim().replace(/\s+/g, ' ').toLocaleLowerCase();
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)